src/main/java/project/bookstore/controller/BookController.java

```diff
@@ -40,19 +41,22 @@ public BookDto getBookById(@PathVariable Long id) {
         return bookService.findBookById(id);
     }
 
+    @PreAuthorize("hasAuthority('ADMIN')")
```

Comment:
Let's change it to `hasRole` everywhere. Investigate the difference between roles and authorities.
```suggestion
    @PreAuthorize("hasRole('ADMIN')")
```